lib/reprocessor.rb

```diff
@@ -152,6 +152,19 @@ def with_id_lines
     end
   end
 
+  def lambda_create_relationships
+    @lambda_create_relationships ||= lambda { |line, progress|
+      id = line.strip
+      e = Bulkrax::Entry.find(id)
+      ::SEEN ||= []
+      if ::SEEN.include?(e.importer.id)
+        e.parser.create_parent_child_relationships
+      else
+        ::SEEN << e.importer.id
+      end
```

Comment:
Can you help me understand this conditional? It looks like we're skipping creating relationships for the first entry from the importer

Reply:
That’s because it’s backwards. It should be !include?  Good catch